.classpath

```diff
@@ -50,11 +50,9 @@
 	<classpathentry kind="lib" path="/jars/lib/jars/logback/logback-classic-1.1.2.jar"/>
 	<classpathentry kind="lib" path="/jars/lib/jars/apache_commons/velocity-1.7.jar"/>
 	<classpathentry kind="lib" path="/jars/lib/jars/apache_commons/commons-lang-2.4.jar"/>
```

Comment:
Needs:
* update documentation describing with what handle service the WSS is compatible (https://ci.kbase.us/services/ws/docs/buildandconfigure.html#service-dependencies) and update the configuration docs (https://ci.kbase.us/services/ws/docs/buildandconfigure.html#handle-manager-url)
* version bump to 0.9.1 in the server class (https://github.com/kbase/workspace_deluxe/blob/develop/src/us/kbase/workspace/WorkspaceServer.java#L111)

Reply:
👍